lib/BloqadeExpr/Project.toml

```diff
@@ -25,6 +25,7 @@ MLStyle = "0.4"
 Unitful = "1"
 YaoAPI = "0.4"
 YaoBlocks = "0.13"
+BloqadeLattices = "0.1.6"
```

Comment:
why do you need a patch version here? all `0.1.x` should be compatible with each other.